include/RAJA/config.hpp.in

```diff
@@ -589,7 +589,7 @@ T * align_hint(T * x)
 #endif
 
 // If we're in CUDA or HIP device code, we can use the unroll pragma
-#if (defined(__CUDA_ARCH__) || defined(__HIP_DEVICE_COMPILE__)) && defined(RAJA_DEVICE_ACTIVE)
+#if (defined(__CUDA_ARCH__) || defined(__HIP_DEVICE_COMPILE__)) && defined(RAJA_GPU_ACTIVE)
```

Comment:
It's unfortunate that `__HIP_DEVICE_COMPILE__` almost makes it seem like we should use `RAJA_GPU_DEVICE_COMPILE_PASS_ACTIVE` here, but we can't change the HIP macro.